### DISCUSSION TITLE
Don't send to homepage after deleting page within fictitious record set

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5843] [Feature Requests]
**Describe the bug**
When in a fictitious record set, deleting a record brings back to home page. This is true even if deleting the newest record.

**To Reproduce**
1. Go to Data Entry
2. Create fictitious record set using `Clone`, `Carry Forward` and/or `Add` buttons
3. Delete a record

**Expected behavior**
Should remain in the record set dialog after deleting a record.

**Screenshots**

https://github.com/specify/specify7/assets/135047322/bedf61f7-5891-4779-aa63-a1cca0a3e81b

**Desktop:**
 - OS: macOS
 - Browser: Chrome
 - Specify 7 Version: v7.9-dev

**Database Name:**
herb_rbge_3_31_23

**Comment by bronwyncombs** [comment]
Tested in earlier versions. This behavior is in `v7.6` so prioritize accordingly

**Comment by emenslin** [comment]
Can recreate in edge (7.9.6)
